startupscript/butane/idle-shutdown.sh

```diff
@@ -46,6 +39,11 @@ LAST_ACTIVE_PROXY="$(get_guest_attribute "last-active/proxy" "0")"
 if [[ "${LAST_ACTIVE}" -lt "${LAST_ACTIVE_PROXY}" ]]; then
     LAST_ACTIVE="${LAST_ACTIVE_PROXY}"
 fi
+# get the latest time between the last boot time and the last active time. The last active timestamp could still be really old when the VM is rebooted and
+# the last active timestamp is not updated yet.
```

Comment:
Since `UP_TIME` is number of seconds since boot and `LAST_ACTIVE` is a timestamp, shouldn't we calculate based on `BOOT_TIME="$((NOW - UP_TIME))"` instead of `UP_TIME` on these two lines?

Reply:
right.. sorry made the change too fast